src/esp/physics/bullet/BulletURDFImporter.cpp

```diff
@@ -0,0 +1,664 @@
+        if ((flags & CUF_USE_SELF_COLLISION_EXCLUDE_ALL_PARENTS) != 0) {
+          cache.m_bulletMultiBody->getLink(mbLinkIndex).m_flags |=
+              BT_MULTIBODYLINKFLAGS_DISABLE_ALL_PARENT_COLLISION;
+        }
+      } else {
+        //					if (canSleep)
+        {
+          if (cache.m_bulletMultiBody->getBaseMass() == 0)
+          //&& cache.m_bulletMultiBody->getNumDofs()==0)
+          {
+            // col->setCollisionFlags(btCollisionObject::CF_KINEMATIC_OBJECT);
+            col->setCollisionFlags(col->getCollisionFlags() |
+                                   btCollisionObject::CF_STATIC_OBJECT);
+            isDynamic = false;
+          }
+        }
+
+        cache.m_bulletMultiBody->setBaseCollider(col);
+      }
+
+      ASSERT(isDynamic != col->isStaticOrKinematicObject(),
+             "Static or Kinematic object erroneously marked dynamic. This "
+             "should not happen.");
+
+      // This group-selection logic isn't very useful. We can't distinguish
+      // between articulated objects here, e.g. cabinets versus robots. Users
+      // will generally have to override the group in their URDFs (see
+      // getCollisionGroupAndMask below). Note use of Noncollidable. By
+      // convention, fixed links should be authored in the URDF as
+      // Noncollidable. Then, we will create fixed rigid bodies, separate from
+      // the multibody, which will be collidable (CollisionGroup::Static) (see
+      // BulletArticulatedObject.cpp).
+      // int collisionFilterGroup = isDynamic ? int(CollisionGroup::Robot)
+      //                                     :
+      //                                     int(CollisionGroup::Noncollidable);
+      int collisionFilterGroup = int(CollisionGroup::Robot);
+
+      int colGroup = 0, colMask = 0;
+      int collisionFlags =
+          getCollisionGroupAndMask(urdfLinkIndex, colGroup, colMask);
+
+      if ((collisionFlags & io::URDF::HAS_COLLISION_GROUP) != 0) {
+        collisionFilterGroup = colGroup;
+      }
+
+      int collisionFilterMask = uint32_t(CollisionGroupHelper::getMaskForGroup(
+          CollisionGroup(collisionFilterGroup)));
+// We don't like overriding the mask in the URDF; we disable support for this.
+// We prefer to only override the group, while still using getMaskForGroup
+// (above) for mask computation.
+#if 0
+      if (collisionFlags & io::URDF::HAS_COLLISION_MASK) {
+        collisionFilterMask = colMask;
+      }
+#endif
+
+      // Mn::Debug{}
+      //    << "addCollisionObject: " << collisionFilterGroup << " , "
+      //    << collisionFilterMask;
+      world1->addCollisionObject(col, collisionFilterGroup,
+                                 collisionFilterMask);
+      // world1->addCollisionObject(col, 2, 1+2);
+      // TODO: fix this collision issue
+
+      // TODO: include the articulated object id here
+      const auto& debugModel = getModel();
+      std::string linkDebugName = "URDF, " + debugModel->m_name + ", link " +
```

Comment:
This local variable is unused. This can be deleted.

For context, see also the old integration with BulletDebugManager: https://github.com/facebookresearch/habitat-sim/blob/articulated-objects-prototype/src/esp/physics/bullet/BulletURDFImporter.cpp#L657